src/main/kotlin/org/rust/ide/formatter/RustFormattingModelBuilder.kt

```diff
@@ -3,19 +3,28 @@ package org.rust.ide.formatter
 import com.intellij.formatting.FormattingModel
 import com.intellij.formatting.FormattingModelBuilder
 import com.intellij.formatting.FormattingModelProvider
+import com.intellij.formatting.SpacingBuilder
 import com.intellij.lang.ASTNode
 import com.intellij.openapi.util.TextRange
 import com.intellij.psi.PsiElement
 import com.intellij.psi.PsiFile
 import com.intellij.psi.codeStyle.CodeStyleSettings
+import org.rust.lang.RustLanguage
+import org.rust.lang.core.psi.RustTokenElementTypes.COMMA
 
 class RustFormattingModelBuilder : FormattingModelBuilder {
     override fun getRangeAffectingIndent(file: PsiFile?, offset: Int, elementAtOffset: ASTNode?): TextRange? = null
 
     override fun createModel(element: PsiElement, settings: CodeStyleSettings): FormattingModel {
-        val block = RustFormattingBlock(element.node)
+        val block = RustBlock(element.node, settings, createSpacingBuilder(settings))
         return FormattingModelProvider.createFormattingModelForPsiFile(element.containingFile, block, settings)
     }
 
+    protected fun createSpacingBuilder(settings: CodeStyleSettings): SpacingBuilder {
+        val rustSettings = settings.getCustomSettings(RustCodeStyleSettings::class.java)
+        val commonSettings = settings.getCommonSettings(RustLanguage)
+        return SpacingBuilder(commonSettings)
+            .after(COMMA).spaceIf(commonSettings.SPACE_AFTER_COMMA)
+            .before(COMMA).spaceIf(commonSettings.SPACE_BEFORE_COMMA)
```

Comment:
I have no strong feeling about this, but I think that providing an option is always worse then just doing the right thing. It's easier to configure a small amount of important flags than to look through the huge pile of trivial knobs. While it may be easy to make something configurable, it's often better to think harder and to figure out the correct solution without the help from the user. 

So, I think that ultimately the Go way of having exactly zero options is the right choice. But unlike Go, Rust does not yet have a singe coding standard, so some config is necessary. `rustfmt` does not allow to change this setting.